database/src/main/java/com/firebase/ui/database/FirebaseArray.java

```diff
@@ -20,25 +20,26 @@
 import com.google.firebase.database.Query;
 
 import java.util.ArrayList;
-import java.util.List;
+import java.util.HashMap;
 
 /**
  * This class implements an array-like collection on top of a Firebase location.
  */
 class FirebaseArray implements ChildEventListener {
-    public interface OnChangedListener {
+    interface OnChangedListener {
```

Comment:
Studio gives me a lint warning saying it's not necessary, which means it's not actually being used outside the package, just like the FirebaseArray class itself.